indico/modules/events/papers/operations.py

```diff
@@ -238,3 +238,16 @@ def update_reviewing_roles(event, users, contributions, role, assign):
                   'Papers unassigned ({})'.format(orig_string(role.title)), session.user, data=log_data)
     db.session.flush()
     logger.info('Paper reviewing roles in event %r updated by %r', event, session.user)
+
+
+def set_deadline(event, deadline, role, enforce):
+    paper_reviewing_settings.set(event, '{}_deadline'.format(role), deadline)
+    paper_reviewing_settings.set(event, 'enforce_{}_deadline'.format(role), enforce)
+    log_data = {'Enforced': enforce}
+    if deadline:
+        log_data['Deadline'] = deadline.isoformat()
```

Comment:
`log_data['Deadline'] = deadline.isoformat() if deadline else None` - no need to spread this over 4 lines